doc/classes/CanvasItem.xml

```diff
@@ -569,13 +569,14 @@
 			The rendering layers in which this [CanvasItem] responds to [Light2D] nodes.
 		</member>
 		<member name="material" type="Material" setter="set_material" getter="get_material">
-			The material applied to textures on this [CanvasItem].
+			The material applied to this [CanvasItem].
 		</member>
 		<member name="modulate" type="Color" setter="set_modulate" getter="get_modulate" default="Color(1, 1, 1, 1)">
-			The color applied to textures on this [CanvasItem].
+			The color applied to this [CanvasItem].
```

Comment:
Perhaps worth saying it explicitly that this property does affect children?
```suggestion
			The color applied to this [CanvasItem]. This property does affect child [CanvasItem]s, unlike [member self_modulate] which only affects the node itself.
```

Reply:
Can be explicit, yes. Just not sure if kinda restating the other property's description is needed. Like wouldn't this be enough:
```suggestion
			The color applied to this [CanvasItem]. This property does affect child [CanvasItem]s (unlike [member self_modulate]).
```
?
(same the other way around for `self_modulate`)

Reply:
I'm fine with that, but if your main concern is duplication, then, as I mentioned in another comment here, it's not bad in itself, as descriptions can sometimes be seen without further context.

Reply:
The problem with duplication is it tends to get out of sync. So I guess I'm trying to avoid it by default (by referring to a single source of truth). But in here it's rather unlikely for this duplicated part to get out of date anyway so this shouldn't be an issue (famous last words). I'm fine with both approaches, the one with cross referencing only somehow seems more natural to me, that's all I guess. :slightly_smiling_face: I'll go with your suggestion.